### ISSUE TITLE
can run it on mac m2?

### ISSUE THREAD

**Issue by joeylin** [anliyuan/Ultralight-Digital-Human#35]
can run it on mac m2?

**Comment by anliyuan** [comment]
Yes. convert weight to onnx, you can run it on any devices

**Comment by joeylin** [comment]
THX, But I mean can I train the model on mac M2 ?

**Comment by anliyuan** [comment]
> THX, But I mean can I train the model on mac M2 ?

yes it can train on M2, just need to put all data and model on cpu.
